### PR TITLE
Fix Android build issues

### PR DESCRIPTION
**Pull request by slawomiradamczyk**
Binary "dlt-test-logstorage" doesn't have a source code and soong fails. This commit removes that binary.

Added two defines "DLT_WRITEV_TIMEOUT_SEC, DLT_WRITEV_TIMEOUT_USEC" that are used by dlt_user_shared.c"

### CODE REVIEW

**Comment by minminlittleshrimp** [comment]
Hello @slawomiradamczyk 
The dlt-test-logstorage should be also upstreamed.
Hello @Bichdao021195 , could you kindly check this?
Regards

**Comment by slawomiradamczyk** [comment]
Well I really don't know how, cause there is no source code for it?

**Comment by minminlittleshrimp** [comment]
No worry @slawomiradamczyk 
Maybe the upstream by our team member missing that, I am working internally for the source code.
Sorry for any inconvenience.
Regards

File: Android.bp [hunk 1]
```diff
@@ -30,6 +30,8 @@ cc_defaults {
         "-DDLT_LIB_USE_UNIX_SOCKET_IPC",
         "-DCONFIGURATION_FILES_DIR=\"/vendor/etc\"",
         "-DDLT_USER_IPC_PATH=\"/dev/socket\"",
+        "-DDLT_WRITEV_TIMEOUT_SEC=1",
+        "-DDLT_WRITEV_TIMEOUT_USEC=0",
```
Comment (michael-methner) [inline Android.bp:34]: Hello @slawomiradamczyk ,
can you remove the configuration change from the Android.bp? I assume that this is unrelated to the the compile issue.

I would then merge this PR, as we are currently not able upstream our changes due to limited team capacity.

Comment (michael-methner) [reply Android.bp:34]: you are right. i missed that one. Thanks for your contribution and patience.

**Review by michael-methner — APPROVED** [review]
